x/profiles/keeper/keeper_chain_link.go

```diff
@@ -0,0 +1,93 @@
+package keeper
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkquery "github.com/cosmos/cosmos-sdk/types/query"
+
+	"github.com/desmos-labs/desmos/x/profiles/types"
+)
+
+// StoreChainLink sotres the given chain link inside the current context.
+// It assumes that the given chain link has already been validated.
+func (k Keeper) StoreChainLink(ctx sdk.Context, link types.ChainLink) error {
+
+	if _, found := k.GetChainLink(ctx, link.ChainConfig.Name, link.Address); found {
+		return fmt.Errorf("chain link already exists")
+	}
+
+	store := ctx.KVStore(k.storeKey)
+	key := types.ChainsLinksStoreKey(link.ChainConfig.Name, link.Address)
+	store.Set(key, k.cdc.MustMarshalBinaryBare(&link))
+	return nil
+}
+
+// GetChainLink returns the chain link corresponding to the given address and the given chain name inside the current context.
+func (k Keeper) GetChainLink(ctx sdk.Context, address string, chainName string) (link types.ChainLink, found bool) {
+	store := ctx.KVStore((k.storeKey))
+
+	bz := store.Get(types.ChainsLinksStoreKey(address, chainName))
+	if bz != nil {
+		k.cdc.MustUnmarshalBinaryBare(bz, &link)
+		return link, true
+	}
+	return types.ChainLink{}, false
```

Comment:
I think it's better to reverse the check here, making it more similar to other code parts: 

```suggestion
	bz := store.Get(types.ChainsLinksStoreKey(address, chainName))
	if bz == nil {
		return types.ChainLink{}, false
	}
	
	k.cdc.MustUnmarshalBinaryBare(bz, &link)
	return link, true
```